### DISCUSSION TITLE
New version ?

### DISCUSSION THREAD

**Discussion by Meita80** [whyboris/Video-Hub-App#739] [General]
Hi,
will there be an updated version to download in the near future ?
Unfortunately I'm not a developer and have absolutely no clue how to compile/build anything from this repository.
Thx,
Meita

**Comment by whyboris** [comment]
Hello Meita,

Thank you for the message! I've been meaning to release a _bugfix_ version for at least a few months, but since I moved to a new house last year I've been very occupied with house projects (and I still have a full time job 😓 ) so VHA releases have been delayed.

I hope to release a _bugfix_ version this spring 🤞 

ps - to build the app from the repository, you only need a few tools installed on your computer: `git` and `node` (both free) - you just have to execute a few commands from a _terminal_ when you have the tools installed. I can give you a hand if you reach out.

**Comment by whyboris** [comment]
It's been a busy year; I am now aiming to release the _bugfix_ version sometime before the end of 2022 🤞

**Comment by whyboris** [comment]
I am hoping to finally release this weekend 🤞 

Was planning on last weekend, in time for February 20th, the _Fifth Anniversary_ of the app's initial launch 🥳 but got stalled when the _Windows_ executable build size was about 250mb instead of about 70mb 😢 -- trying to debug why the size ballooned so much.

When I re-built on my wife's laptop (clean dev environment) the build was 150mb which is acceptable, but still double of what I'd prefer for a final release.

Hoping to figure it out this weekend ❤️

**Comment by whyboris** [comment]
🥳 Finally released the `3.2.0` version 🚀 thank you all for the patience.

https://github.com/whyboris/Video-Hub-App/releases/tag/v3.2.0

ps - I have ideas for getting the build size down, so I'll try it out sometime this week 🤞
